src/CountUp.tsx

```diff
@@ -1,9 +1,22 @@
 import PropTypes from 'prop-types';
```

Comment:
I think that prop types should be removed

Reply:
Oh yeah for sure. Feel free to make this change to my PR branch as I probably can't circle back until midweek.

Reply:
Removed prop types.